astroquery/alma/tapsql.py

```diff
@@ -69,16 +69,21 @@ def _gen_pos_sql(field, value):
                     dec_min = -90
                 if dec_max is None:
                     dec_max = 90
-                min_pt = coord.SkyCoord(ra_min, dec_min, unit=u.deg,
-                                        frame=frame)
-                max_pt = coord.SkyCoord(ra_max, dec_max, unit=u.deg,
-                                        frame=frame)
-                result += \
-                    "(INTERSECTS(RANGE_S2D({},{},{},{}), s_region) = 1)".\
-                    format(min_pt.icrs.ra.to(u.deg).value,
-                           max_pt.icrs.ra.to(u.deg).value,
-                           min_pt.icrs.dec.to(u.deg).value,
-                           max_pt.icrs.dec.to(u.deg).value)
+                min_pt = coord.SkyCoord(ra_min, dec_min, unit=u.deg)
+                ra_min, dec_min = min_pt.ra.value, min_pt.dec.value
+                max_pt = coord.SkyCoord(ra_max, dec_max, unit=u.deg)
+                ra_max, dec_max = max_pt.ra.value, max_pt.dec.value
+                if frame == 'galactic':
+                    # intersect with s_region is too complicated. ALMA indicated that
+                    # the use of gal_longitude and gal_latitude is good enough
+                    # approximation in this less comon use case
+                    result += ('gal_longitude>={} AND gal_longitude<={} AND '
+                               'gal_latitude>={} and gal_latitude<={}').format(
```

Comment:
minor typo in the comment.  Is the query case-sensitive?  the third `and` is lower-case.
```suggestion
                    # approximation in this less common use case
                    result += ('gal_longitude>={} AND gal_longitude<={} AND '
                               'gal_latitude>={} and gal_latitude<={}').format(
```